AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_AtomEditorComponents_DecalAdded.py

```diff
@@ -3,132 +3,137 @@
     Creation and deletion undo/redo should also work.
 
     Test Steps:
-    1) Creation of entity with Decal component.
-    2) UNDO the entity creation.
-    3) REDO the entity creation.
-    4) Enter/Exit game mode.
-    5) Hide test.
-    6) Visible test.
-    7) Add Material component.
-    8) Delete Decal entity.
-    9) UNDO deletion.
-    10) REDO deletion.
-    11) Look for errors.
+    1) Creation of Decal entity.
+    2) Add Decal component to Decal entity.
+    3) UNDO the entity creation and component addition.
+    4) REDO the entity creation and component addition.
+    5) Enter/Exit game mode.
+    6) Hide test.
+    7) Visible test.
+    8) Set Material property on Decal component.
+    9) Delete Decal entity.
+    10) UNDO deletion.
+    11) REDO deletion.
+    12) Look for errors.
 
     :return: None
     """
     import os
 
     import azlmbr.asset as asset
     import azlmbr.bus as bus
-    import azlmbr.entity as entity
     import azlmbr.editor as editor
     import azlmbr.legacy.general as general
     import azlmbr.math as math
-    import azlmbr.paths
 
-    from editor_python_test_tools import hydra_editor_utils as hydra
-    from editor_python_test_tools.utils import Report
-    from editor_python_test_tools.utils import TestHelper as helper
-    from editor_python_test_tools.utils import Tracer
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import Report, Tracer, TestHelper as helper
 
     with Tracer() as error_tracer:
-        # Wait for Editor idle loop before executing Python hydra scripts.
+        # Test setup begins.
+        # Setup: Wait for Editor idle loop before executing Python hydra scripts then open "Base" level.
         helper.init_idle()
-
-        # Delete all existing entities initially for test setup.
-        search_filter = azlmbr.entity.SearchFilter()
-        all_entities = entity.SearchBus(azlmbr.bus.Broadcast, "SearchEntities", search_filter)
-        editor.ToolsApplicationRequestBus(bus.Broadcast, "DeleteEntities", all_entities)
-
-        # 1. Creation of entity with Decal component.
-        decal = "Decal"
-        decal_entity = hydra.Entity(f"{decal}")
-        decal_entity.create_entity(math.Vector3(512.0, 512.0, 34.0), [decal])
-        has_component = hydra.has_components(decal_entity.id, [decal])
-        Report.result(Tests.decal_creation, decal_entity.id.IsValid())
-        Report.critical_result(Tests.decal_component, has_component)
-
-        # 2. UNDO the entity creation.
-        general.undo()
-        Report.result(Tests.creation_undo, decal_entity.id.IsValid())
-
-        # 3. REDO the entity creation.
-        general.redo()
-        Report.result(Tests.creation_redo, decal_entity.id.isValid())
-
-        # 4. Enter/Exit game mode.
+        helper.open_level("Physics", "Base")
+
+        # Test steps begin.
+        # 1. Creation of Decal entity.
+        decal = "Decal (Atom)"
```

Comment:
it's used in step 1 also (also changing `decal` to `decal_name` in the next update):
```
        decal_name = "Decal (Atom)"
        decal_entity = EditorEntity.create_editor_entity_at(math.Vector3(512.0, 512.0, 34.0), f"{decal_name}")
```

Reply:
i see the argument for a constant but i think its easier to have it near the 2 steps its used in rather than scroll to the top of the module, besides i dont think its used in many other locations other than those 2 steps